packages/ui-bindings/EconomicEvent/EconomicEvent.tsx.ts

```diff
@@ -0,0 +1,5 @@
+
+export default interface EconomicEvent {
```

Comment:
this is not an accurate representation of an economic event. consider also adding a fragment to this file to use in your queries

Reply:
"This is not an accurate representation" - fields are missing

Reply:
Interface temporarily removed until the rest of the types are defined
:x: